core/src/components/select/select.ionic.scss

```diff
@@ -181,6 +182,18 @@
   color: globals.$ion-primitives-neutral-500;
 }
 
+// Focus
+// ---------------------------------------------
+:host(.ion-focused) .select-wrapper-inner {
+  @include globals.focused-state(null, null, var(--focus-ring-color));
+}
+
+// Invalid
+// ---------------------------------------------
+:host(.ion-invalid) {
+  --focus-ring-color: #{globals.$ion-border-focus-error};
+}
```

Comment:
This needs to follow the structure of `md` that uses `--highlight-color-invalid`:
- https://github.com/ionic-team/ionic-framework/blob/ef9a8ef550b771c5c8154f1db95343e85396d5e0/core/src/components/select/select.common.scss#L238-L240
- https://github.com/ionic-team/ionic-framework/blob/ef9a8ef550b771c5c8154f1db95343e85396d5e0/core/src/components/select/select.md.outline.scss#L25-L28
- https://github.com/ionic-team/ionic-framework/blob/ef9a8ef550b771c5c8154f1db95343e85396d5e0/core/src/components/select/select.md.solid.scss#L30-L33

Reply:
Done!